manifests/params.pp

```diff
@@ -61,4 +61,6 @@
   $configs = {}
 
   $plugins = {}
+
+  $purge_config_dir = true
```

Comment:
I would default this to `false` since it was not the default behavior previously. Purging is destructive and i don't want to cause any issues or outages for someone in their environment.

```suggestion
  $purge_config_dir = false
```